### PR TITLE
Project suggestion for PGD: user management system overhaul

### PR DESCRIPTION
**Pull request by mathuin**
The user management system could use more attention, especially since it's being used by the PIs and import/promote will add additional responsibilities for certain users.

Also, the 2014 thing bothered me so I fixed it. :-)

### CODE REVIEW

File: source/gsoc/ideas2015.rst [hunk 1]
```diff
@@ -119,6 +119,8 @@ file formats and support for very large proteins.
 
 This is an excellent project for anyone with an interest in bioinformatics or open source science.
 
+One specific area of the PGD that needs improvement is the user management system.  Users should be able to store and manipulate their searches, including comparing historical results to current results.  There is some additional functionality planned for admin users with regard to database management as well.
+
```
Comment (ramereth) [inline source/gsoc/ideas2015.rst]: Can you please line wrap this?

Comment (mathuin) [inline source/gsoc/ideas2015.rst]: I wrapped the lines here and updated the diff.

**Comment by mathuin** [comment]
Any other comments?  I would like to get this merged quickly as mentoring organizations have a submission deadline and students will be seeing the page soon!

**Comment by Kennric** [comment]
+1 looks good to me, and good catch on those dates

**Pull request merged**